keyboards/handwired/onekey/keymaps/joystick/keymap.c

```diff
@@ -0,0 +1,25 @@
+#include QMK_KEYBOARD_H
+
+#include "joystick.h"
+
+#ifndef ADC_PIN
+#    define ADC_PIN F6
+#endif
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+  LAYOUT( JS_BUTTON0 )
```

Comment:
This does not compile after some recent changes to `handwired/onekey` — the `LAYOUT` macro was renamed:
```c
  LAYOUT_ortho_1x1( JS_BUTTON0 )
```